test/e2e-gke.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source ${PWD}/cluster
+
+readonly IMAGE_TAG=${TEST_IMAGE_TAG}
+readonly IMAGE_REPOSITORY="gcr.io/kubernetes-charts-ci/test-image"
+readonly REPO_ROOT="${REPO_ROOT:-$(git rev-parse --show-toplevel)}"
+
+main() {
+    if [[ $(git remote | grep k8s) = '' ]]; then
+      git remote add k8s ${CHARTS_REPO}
+    fi
+    git fetch k8s master
+
+    echo $GCLOUD_SERVICE_KEY_CHARTS_CI | base64 --decode -i > ${PWD}/gcloud-service-key.json
```

Comment:
Where is `GCLOUD_SERVICE_KEY_CHARTS_CI` managed? In Jenkins?

Reply:
it is TravisCI env var